lib/licenses.nix

```diff
@@ -760,6 +760,11 @@ lib.mapAttrs (n: v: v // { shortName = n; }) ({
     # channel and NixOS images.
   };
 
+  unicode-dfs-2015 = spdx {
```

Comment:
Not completely sure if adding this here is worth it but given that `unicode-dfs-2016` is already there it might make sense.